### DISCUSSION TITLE
Authorization Error in GitHub Classroom

### DISCUSSION THREAD

**Discussion by MSMCDigitalLearning** [community/community#106310] [GitHub Education]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

GitHub Classroom was added to Moodle 4.1 as a preconfigured external tool, but does not appear to be working. All [configuration steps](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/register-a-learning-management-system-with-github-classroom#configuring-moodle-for-github-classroom) had been completed successfully. 

**However, when an instructor attempts to "select content" for the external tool, it prompts them to select a classroom to link. Clicking on the link that is prompted, it says a classroom has already been linked (i.e., from a previous attempt).**

![image](https://github.com/community/community/assets/158207046/d912807f-21ce-4b04-9931-873f9d0a2abb)

**Also, when a student attempts to enter the external tool, the following error is seen:**

![image](https://github.com/community/community/assets/158207046/0acf956e-0fe9-4731-9975-d05cba4c6e53)

**Please advise.**

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MSMCDigitalLearning ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
